src/configfile.c

```diff
@@ -221,6 +222,11 @@ static int config_insert(server *srv) {
 		s->use_ipv6      = 0;
 		s->set_v6only    = 1;
 		s->defer_accept  = 0;
+#ifdef SO_ACCEPTFILTER
+		s->use_acceptfilter = 1;
+#else
+		s->use_acceptfilter = 0;
+#endif
```

Comment:
Since this is a BSD-specific flag, do the BSD core developers recommend that the default be enabled or disabled?  (Previously behavior in lighttpd was always enabled, so thank you for attempting to preserve that behavior.)